### PR TITLE
ci: update Gemfile

### PR DESCRIPTION
**Pull request by daks**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
-source 'https://rubygems.org'
+source "https://rubygems.org"
+
+gem "test-kitchen"
```
Comment (myii) [inline Gemfile]: This will be pulled in by the others so can be removed.

File: Gemfile [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
+gem "kitchen-docker"
+gem "kitchen-salt"
+gem "kitchen-inspec"
```
Comment (myii) [inline Gemfile]: Can we get the version numbers?

```Gemfile
source "https://rubygems.org"

gem 'kitchen-docker', '>= 2.9'
gem 'kitchen-salt', '>= 0.6.0'
gem 'kitchen-inspec', '>= 1.1'
```

**Comment by myii** [comment]
@daks Feel free to ignore the comments above if there's something different in this repo.  But we got burned badly the last time with old versions of `test-kitchen` being pulled when we didn't have the version numbers in place.

File: Gemfile (hunk 2 above)
Comment (daks) [reply Gemfile]: of course :)

**Comment by myii** [comment]
@daks Thanks, I'll merge this when the tests pass.

**Pull request merged**